src/query/string_opfunc.c

```diff
@@ -8076,7 +8079,12 @@ db_get_string_length (const DB_VALUE * value)
   str = value->data.ch.medium.buf;
   length = size = value->data.ch.medium.size;
   codeset = (INTL_CODESET) value->data.ch.medium.codeset;
+  DB_VALUE *p = (DB_VALUE *) value;
 
+  if (p->data.ch.medium.length != -1)
+    {
+      return p->data.ch.medium.length;
+    }
```

Comment:
How about this?
```
int
db_get_string_length (const DB_VALUE * value)
{
  DB_CONST_C_CHAR str;
  int size;
  INTL_CODESET codeset;
  int length = 0;

  if (value->data.ch.medium.length != -1)
    {
      return value->data.ch.medium.length;
    }

  str = value->data.ch.medium.buf;
  length = size = value->data.ch.medium.size;
  codeset = (INTL_CODESET) value->data.ch.medium.codeset;
  ...
```

Reply:
Based on this opinion, the code was modified: The commit message is
'(DB_VALUE*)p->data => value->data'